pkg/local_object_storage/writecache/writecache.go

```diff
@@ -99,6 +97,12 @@ func New(opts ...Option) Cache {
 		opts[i](&c.options)
 	}
 
+	// Make the LRU cache contain which take approximately 3/4 of the maximum space.
+	// Assume small and big objects are stored in 50-50 proportion.
+	c.maxFlushedMarksCount = int(c.maxCacheSize/c.maxObjectSize+c.maxCacheSize/c.smallObjectSize) * 3 / 8
```

Comment:
can we write it as `int(...+...) / 2 * 3 / 4` to make it easier for a reader?

Reply:
Fixed